protopipe/pipeline/event_preparer.py

```diff
@@ -236,17 +243,18 @@ def __init__(
             )
         )
 
-        # Configuration for the camera calibrator
+        # Configuration for the image extractor
 
         cfg = Config()
+        cfg.TwoPassWindowSum.apply_integration_correction = self.apply_integration_correction
 
         extractor = TwoPassWindowSum(config=cfg, subarray=subarray)
```

Comment:
You can just pass the option in here. Since you are not using a ctapipe config file, you do not need to construct a Config() object, just pass the options in (making Config() objects was only necessary for very old versions of ctapipe):

```py3
extractor = TwoPassWindowSum(
    subarray=subarray, 
    apply_integration_correction=self.apply_integration_correction
)
```

Reply:
done